react-function-components/src/index.jsx

```diff
@@ -7,4 +7,4 @@ function CustomButton() {
 
 const container = document.querySelector('#root');
 const root = ReactDOM.createRoot(container);
-root.render(CustomButton());
+root.render(CustomButton);
```

Comment:
This actually doesn't work. Be sure to create a React element with the component as the type.

```jsx
root.render(<CustomButton />);
```

Reply:
Alright!